compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -815,8 +815,8 @@ error.checked.expr.invalid.usage.no.error.type.rhs=\
 error.checked.expr.invalid.usage.only.error.types.rhs=\
   invalid usage of the ''{0}'' expression operator: all expression types are equivalent to error type
 
-error.checked.expr.no.error.return.in.encl.invokable=\
-  invalid usage of the ''check'' expression operator: no error type return in enclosing invokable
+error.checked.expr.no.matching.error.return.in.encl.invokable=\
+  invalid usage of the ''check'' expression operator: no matching error return type(s) in the enclosing invokable
```

Comment:
should we say function here instead of `invokable`? since we only have functions now.

Reply:
This is the same error used with worker returns too though. Is it OK to say function there also?

```ballerina
public function main() {
    worker w1 {
        check foo();
    }
}

function foo() returns error? {
    return error("err");
}
```

Error with the current changes:
```cmd
$ ballerina run qwe.bal 
Compiling source
        qwe.bal
error: .::qwe.bal:3:9: invalid usage of the 'check' expression operator: no matching error return type(s) in the enclosing invokable
```

Reply:
Ok, Got it. Let's address this issue separately then.